src/NzbDrone.Core/Datastore/Migration/120_blacklist_language.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using FluentMigrator;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Common.Serializer;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(120)]
+    public class blacklist_language : NzbDroneMigrationBase
```

Comment:
Why 4 separate migrations all numbered somewhat randomly? I think we could get away with 2, one that adds language profiles and one that add language it to the appropriate tables.